compiler/tests-common-new/tests-gen/org/jetbrains/kotlin/test/runners/codegen/IrBlackBoxCodegenTestGenerated.java

```diff
@@ -32401,6 +32401,12 @@ public void testAnnotatedAssignment() throws Exception {
             runTest("compiler/testData/codegen/box/operatorConventions/annotatedAssignment.kt");
         }
 
+        @Test
+        @TestMetadata("assignOperatorOverload.kt")
+        public void testAssignOperatorOverload() throws Exception {
+            runTest("compiler/testData/codegen/box/operatorConventions/assignOperatorOverload.kt");
+        }
+
```

Comment:
These are generated source files, right? Why are these committed to VCS?

Reply:
Tests are generated, but you can still run tests manually. So I guess to not regenerate tests on every run which might be slow, they keep it in VCS.